tests/sanity/ignore-2.10.txt

```diff
@@ -1,4 +1,3 @@
-# (c) Copyright IBM Corp. 2020,2021
 docs/make.bat line-endings!skip # Windows batch file requires windows line endings
```

Comment:
Did these copyright headers break something?

Reply:
If you run the full build with these in, it comes up with an error that you can't have a comment as the first thing in these files and fails so I had to remove them. Searched around the error it produced at the time and couldn't find a workaround to leave them in